#Pose of the tag relative
geometry_msgs/PoseStamped pose
int32 tag_id
